src/LobbyServer.hs

```diff
@@ -119,26 +119,29 @@ getGamesList remoteGames = do
   return $ map fst gameList
 
 -- |Lets a player join a 'LobbyGame'
-playerJoinGame :: Server ConcurrentClientList -> Server GamesList -> String -> Server ()
+playerJoinGame :: Server ConcurrentClientList -> Server GamesList -> String -> Server Bool
```

Comment:
Maybe a little comment clarifying that String is gameID. Like it's stated in the LobbyAPI